src/lib.rs

```diff
@@ -21,6 +21,52 @@ use std::ffi::CString;
 
 pub use sapp::{rand, RAND_MAX};
 
+#[cfg(not(target_arch = "wasm32"))]
+pub mod console {
+    pub fn debug(msg: &str) {
+        panic!("console::debug only implemented for wasm32 target");
+    }
+    
+    pub fn log(msg: &str) {
+        panic!("console::log only implemented for wasm32 target");
+    }
+
+    pub fn info(msg: &str) {
+        panic!("console::info only implemented for wasm32 target");
+    }
+
+    pub fn warn(msg: &str) {
+        panic!("console::warn only implemented for wasm32 target");
+    }
+    
+    pub fn error(msg: &str) {
+        panic!("console::error only implemented for wasm32 target");
+    }
+}
+
+#[cfg(target_arch = "wasm32")]
+pub mod console {
```

Comment:
Hmmm, console modue is actually a pretty good idea - make platform independent terminal access for all the platforms. 
But maybe it is a bit outside of the scope of this PR?

Reply:
Ah, I was doing this more to prevent non-wasm32 targets from failing to compile (e.g. when running tests, similar to https://github.com/not-fl3/miniquad/commit/bf56678ed881bf5079befbde121e951f923c7dfa)

It _could_ be used to make it platform independent...

The problem is that `sapp-console-log` doesn't have a way to reach inside of `miniquad` and use the `sapp-wasm` crate... or does it? Can I do `extern crate miniquad::sapp_wasm` from within `sapp-console-log`?